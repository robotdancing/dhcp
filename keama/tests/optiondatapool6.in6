# option-data in DHCPv6 pool config

# authoritative is mandatory
authoritative;

# empty configs are not accepted by Kea
default-lease-time 1800;

# subnet declaration
subnet6 2001::/64 {
    # pool declaration
    pool6 {
        # for a silly reason option-data is not allowed in DHCPv4 pools
	# but allowed in DHCPv6 pools
	option dhcp6.domain-search example.com, example.org;
    }
}
