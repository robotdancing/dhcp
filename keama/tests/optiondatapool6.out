{
  # option-data in DHCPv6 pool config
  # authoritative is mandatory
  /// This configuration declares some subnets but has no interfaces-config
  /// Reference Kea #5256
  "Dhcp6": {
    "subnet6": [
      # subnet declaration
      {
        "subnet": "2001::/64",
        "pools": [
          # pool declaration
          {
            # avoid empty pool
            "pool": "2001::100 - 2001::200",
            "option-data": [
              # for a silly reason option-data is not allowed in DHCPv4 pools
              # but allowed in DHCPv6 pools
              {
                "space": "dhcp6",
                "name": "domain-search",
                "code": 24,
//              "original-data": "\"example.com\", \"example.org\"",
                "data": "example.com, example.org"
              }
            ]
          }
        ]
      }
    ]
  }
}
